src/storage/secret.rs

```diff
@@ -1,10 +1,14 @@
 use crate::error::AppError;
+use crate::rocket::futures::TryStreamExt;
+use log::info;
 use mongodb::bson::{doc, Bson};
+use rocket::tokio::process;
 use serde::{Deserialize, Serialize};
 use sha3::{Digest, Keccak256};
 use uuid::Uuid;
 
 use super::storage::Storage;
+use crate::AttemptCountRule;
```

Comment:
Talking about unused import, we also have these warnings:
```
 --> src/handlers/secret.rs:5:14
  |
5 | use rocket::{Response, State};
  |              ^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: unused import: `crate::rocket::futures::TryStreamExt`
  --> src/handlers/secret.rs:10:5
   |
10 | use crate::rocket::futures::TryStreamExt;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: unused import: `log::info`
 --> src/storage/secret.rs:3:5
  |
3 | use log::info;
  |     ^^^^^^^^^

warning: unused import: `rocket::tokio::process`
 --> src/storage/secret.rs:5:5
  |
5 | use rocket::tokio::process;
  |     ^^^^^^^^^^^^^^^^^^^^^^

warning: unused import: `crate::AttemptCountRule`
  --> src/storage/secret.rs:11:5
   |
11 | use crate::AttemptCountRule;
   |     ^^^^^^^^^^^^^^^^^^^^^^^

warning: variant `DuplicateEntryError` is never constructed
 --> src/error.rs:4:5
  |
2 | pub enum AppError {
  |          -------- variant in this enum
3 |     AlreadyGuessed,
4 |     DuplicateEntryError,
  |     ^^^^^^^^^^^^^^^^^^^
  |
  = note: `AppError` has a derived impl for the trait `Debug`, but this is intentionally ignored during dead code analysis
  = note: `#[warn(dead_code)]` on by default
```

Reply:
All these unused been removed